native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/RestAPIExplorerViewController.m

```diff
@@ -465,15 +465,15 @@ -(BOOL)textFieldShouldReturn:(UITextField *)textField {
 
 #pragma mark - SFRestDelegate
 
-- (void)request:(SFRestRequest *)request didLoadResponse:(id)dataResponse rawResponse:(NSURLResponse *)rawResponse {
+- (void)request:(SFRestRequest *)request didLoadResponse:(id)dataResponse {
```

Comment:
for me to test ;-)
Also one should only really use the new signature if one plans to actually use the raw response

Reply:
Fair enough.